.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+**Do you want to request a *feature* or report a *bug*?**
+
+**What is the current behavior?**
+
+**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
+
+**What is the expected behavior?**
+
+**Which OS, version of Rust, PostgreSQL and IPFS are you using?**
```

Comment:
This seems excessive, most bugs will be independent from the version of these dependencies. What do you think of removing this and just ask when relevant?

Reply:
Sure!